webpack.config.js

```diff
@@ -44,9 +43,11 @@ module.exports = (env, argv) => {
         // when working on the app but adds significant runtime overhead
         // We want to use the React production build but not compile the whole
         // application to productions standards
-        nodeEnv = "production";
+        argv.mode = "production";
```

Comment:
can we document why we need to modify the process `argv`?

Reply:
It's mostly because we're doing it in the wrong way. 😛 
We should use `NODE_ENV` instead of some CI env, but we're doing CI env to override NODE_ENV instead.

Reply:
I assume we don't mean to set the value of argv.mode twice here?

Reply:
Now that I'm having a second look at that I'm starting to wonder what's going on here. @turt2live it seems you turned it off, is this still relevant and necessary? We have proper sourcemaps anyways.

Reply:
At the time of us doing this there was a difference between us using `NODE_ENV` and what we actually wanted to ship, so even though it was a development NODE_ENV we didn't want the bloated sourcemaps. If our requirements are aligned on NODE_ENV now, we can switch to that.

The only other reason was to let developers build production-ready builds on their machines, but we haven't done that in at least a year.

Reply:
We can still turn it on by making `NODE_ENV='production' yarn build`